integrations/access/accessrequest/message.go

```diff
@@ -80,11 +81,22 @@ func MsgFields(reqID string, reqData pd.AccessRequestData, clusterName string, w
 	if len(reqData.User) > 0 {
 		msgFieldToBuilder(&builder, "User", reqData.User)
 	}
-	if len(reqData.Roles) > 0 {
-		msgFieldToBuilder(&builder, "Role(s)", strings.Join(reqData.Roles, ","))
+	if len(reqData.LoginsByRole) > 0 {
+		for role, logins := range reqData.LoginsByRole {
+			var loginStr string
+			if len(logins) > 0 {
+				loginStr = strings.Join(logins, ", ")
+			} else {
+				loginStr = "none"
```

Comment:
Can you please add a test for this case?
I wonder if we could be more explicit about this scenario, maybe by using italic style for the `none` word: _none_